frontend-react/src/pages/BasicSettingEditPage.tsx

```diff
@@ -364,7 +375,7 @@ export default function BasicSettingEditPage() {
           </ul>
           {ErrorList([...(formState.error ? [formState.error] : []), ...errors, ...actionState.errors])}
           <div className="text-center my-5">
-            <Button type="submit" variant="primary" size="sm" className="mr-4">更新</Button>
+            <Button type="submit" variant="primary" size="sm" className="mr-4" disabled={!isFormValid()}>更新</Button>
```

Comment:
細かいですがisFormValid()は戻りbooleanのメソッドに見えるので!!は不要そう
```suggestion
            <Button type="submit" variant="primary" size="sm" className="mr-4" disabled={isFormValid()}>更新</Button>
```